src/onetime/entrypoints/web/web_flask/app.py

```diff
@@ -0,0 +1,105 @@
+from flask import Flask
+from flask_cors import CORS
+from werkzeug.middleware.proxy_fix import ProxyFix
+from onetime.entrypoints.web.web_flask.containers import Container
+from cryptography.fernet import Fernet
+from uuid import uuid4
+
+
+app = Flask(__name__)
```

Comment:
Please use the Flask application factory.
A similar approach as shown here:
https://github.com/ShahriyarR/hexagonal-flask-blog-tutorial/blob/main/src/blog/adapters/entrypoints/app/application.py#L9